wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -248,6 +248,24 @@ public void load(Device device) throws ModelException, IOException {
             if (engine == null) {
                 engine = m.getNDManager().getEngine();
             }
+
+            if (models.isEmpty()) {
```

Comment:
why are we checking for this condition?

Reply:
This models is specifically a map from device to model. So, it checks that it is empty to ensure that this loading only happens on the first device load instead of every new device you add the model to